crates/bevy_utils/src/label.rs

```diff
@@ -53,48 +53,55 @@ where
 ///
 /// ```
 /// # use bevy_utils::define_label;
-/// define_label!(MyNewLabelTrait);
+/// define_label!(MyNewLabelTraitId, MyNewLabelTrait);
```

Comment:
Can we collapse this back down to a single argument? I can't imagine this will ever be called in a case where you care about breaking convention on the `*Id`.